apps/iOS-WKWebView-sample/WkWebView DemoUITests/WkWebView_DemoUITests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  WkWebView_DemoUITests.swift
+//  WkWebView DemoUITests
+//
+//
+
+import XCTest
+
+class WkWebView_DemoUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+
+
+    func joinMeetingTest() throws {
+        let app = XCUIApplication()
+        app.launch()
+        app.buttons["Go to WebView"].tap()
+        let webView = app.webViews
+        _ = webView.textFields.element(boundBy: 0).waitForExistence(timeout: 10)
+        let meetingTitleField = webView.textFields.element(boundBy: 0);
+        meetingTitleField.tap()
+        let meetingTitle = UUID().uuidString
+        meetingTitleField.typeText(meetingTitle)
+        let nameField = webView.textFields.element(boundBy: 1)
+        nameField.tap()
+        let attendeeName = UUID().uuidString
+        nameField.typeText(attendeeName)
+        app.toolbars.buttons["Done"].tap()
+        webView.buttons["Continue"].tap()
+        _ = app.alerts.element.waitForExistence(timeout: 10)
+        app.alerts.element.buttons["Allow"].tap()
+        _ = webView.buttons["Join"].waitForExistence(timeout: 10)
+        webView.buttons["Join"].tap()
+        webView.buttons["Toggle microphone"].tap()
+        webView.buttons["Toggle camera"].tap()
```

Comment:
What does it test? I think you would at least need something like `XCTAssertTrue` or some verification.

Reply:
Its mainly testing that we can join the meeting - if any of the .tap() methods fail, then this test will fail. I added an assert at the end to make sure we end up in the meeting view.